unime/src/i18n/en/index.ts

```diff
@@ -189,26 +191,60 @@ const en = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
+      NAVBAR_TITLE: 'Credential info',
+      NAME: '',
+    },
+    BADGE: {
+      NAVBAR_TITLE: 'Badge info',
+      DESCRIPTION: 'Description',
+      METADATA: 'Metadata',
+    },
     EMPTY_CREDENTIALS: {
       TITLE: "It's a bit quiet in here",
       SUBTITLE: 'Why not add some credentials to start your new digital me?',
     },
     FAVORITES: 'My favorites',
   },
   ACTIVITY: {
+    BOTTOM_NAVIGATION_TITLE: 'Activity',
     NAVBAR_TITLE: 'Connected',
     TABS: {
       CONNECTIONS: 'Connections',
       TIMELINE: 'Timeline',
     },
   },
+  SCAN: {
+    BOTTOM_NAVIGATION_TITLE: 'Scan',
+    TITLE_1: 'Scan a',
+    TITLE_2:'QR Code',
+    SUBTITLE: 'Bring a QR Code into view of this screen to start an interaction.',
+    PERMISSION_1: 'No permissions to',
+    PERMISSION_2: 'access the camera',
+    CONNECTION_OFFER: {
+      NAVBAR_TITLE: 'Credential Offer',
+      DESCRIPTION: 'is offering you the following credentials',
+    },
+    CONNECTION_REQUEST: {
+      NAVBAR_TITLE: 'Credential Request',
+      TITLE_1: 'New connection',
+      TITLE_2: 'Only accept new connections that you recognize and trust',
+      URL: 'URL',
```

Comment:
Although it is consitent to translate this, I don't think we need to translate `URL`.